core/domain/opportunity_services.py

```diff
@@ -239,8 +239,15 @@ def update_opportunity_with_updated_exploration(exp_id):
 
     exploration_opportunity_summary.validate()
 
-    _save_multi_exploration_opportunity_summary(
-        [exploration_opportunity_summary])
+    # After all available content has been translated, delete the opportunity.
+    if (
+            exploration_opportunity_summary.translation_counts >=
```

Comment:
Maybe create a method in the domain class which will check this? (Not ExplorationOpportunity model represents translation and voiceover opportunity in an exploration!)